ui/ui-components/pages/app/[id]/edit.tsx

```diff
@@ -365,7 +365,8 @@ export default function Page(props) {
                   </Alert>
                 ) : ranTest ? (
                   <Alert variant="warning">
-                    <strong>Test Failed</strong> {testResult.error}
+                    <strong>Test Failed</strong>{" "}
+                    {testResult.error || testResult.message}
                   </Alert>
```

Comment:
This API probably could be better.  The case you added means that there is both an Error and Message set on the test... is that common?

Reply:
There is no such thing as an `error` actually. Plugins only send `message`. I don't know where `error` came from, but it wasn't in the scope of making this plugin to mess with that.